SwiftReflector/SwiftXmlReflection/TypeSpec.cs

```diff
@@ -353,7 +356,7 @@ protected override bool LLEquals (TypeSpec other, bool partialNameMatch)
 		}
 		public string NameWithoutModule {
 			get {
-				return HasModule ? Name.Substring (Name.IndexOf ('.') + 1) : Name;
+				return Name.Contains ('.') ? Name.Substring (Name.IndexOf ('.') + 1) : Name;
```

Comment:
minor: there's (sadly) no overload for `Constains` that accept characters (it will become a string), `IndexOf` would work on a `char`

Reply:
👍